update.c

```diff
@@ -187,6 +191,89 @@ do_run_dialog (GtkListStore *list_store,
 }
 
 static void
+update_background_image_path (XdgDirEntry *old_entries,
+                              XdgDirEntry *new_entries)
+{
+  GSettings *settings = NULL;
```

Comment:
Nit: no need to separately initialize this to `NULL`, as you use it just below here.